src/docfx/data/docs/trusted-domains.json

```diff
@@ -0,0 +1,191 @@
+{
+  "a": [
```

Comment:
can we communicate with Ke and Megan on this list before it goes to live?

Reply:
Yes, they are already asking for this list in feature comments.